src/index.js

```diff
@@ -30,23 +20,15 @@ module.exports = (/* options */) => ({
   // Fix sourcemaps
   result(stylelintResult, filepath) {
     const lineCorrection = sourceMapsCorrections[filepath]
-    const newWarnings = stylelintResult.warnings.reduce((prevWarnings, warning) => {
-      if (ignoredRules.includes(warning.rule)) return prevWarnings
-      const correctedWarning = Object.assign(warning, {
+    const warnings = stylelintResult.warnings.map(warning =>
+      Object.assign(warning, {
         // Replace "brace" with "backtick" in warnings, e.g.
         // "Unexpected empty line before closing backtick" (instead of "brace")
         text: warning.text.replace(/brace/, 'backtick'),
         line: lineCorrection[warning.line]
       })
-      prevWarnings.push(correctedWarning)
-      return prevWarnings
-    }, [])
-
-    if (newWarnings.length === 0) {
-      // eslint-disable-next-line no-param-reassign
-      stylelintResult.errored = false
-    }
+    )
 
-    return Object.assign(stylelintResult, { warnings: newWarnings })
+    return Object.assign(stylelintResult, { warnings })
```

Comment:
As we currently mutate `warning` in the `.map` above this is actually unnecessary right? I think with current mutations this is actually the same as just returning `stylelintResult`. I understand that's not as clear though, so maybe I'd recommend just not mutating and following my suggestion above and then also not mutating here? As in again add a first argument `{}`.